sandbox/web/tests/loader/pauseafter/ios.ts

```diff
@@ -0,0 +1,30 @@
+/// <reference path="../../../../../dist/Excalibur.d.ts"/>
+
+var game = new ex.Engine({
+   canvasElementId: 'game',
+   displayMode: ex.DisplayMode.FullScreen
+});
+
+var jump = new ex.Sound('../../../sounds/jump.mp3', '../../../sounds/jump.wav'); 
+
+// if iOS, use PauseAfterLoader
+var iOS = /iPad|iPhone|iPod/.test(navigator.userAgent) && !(<any>window).MSStream;
```

Comment:
I could be persuaded to put this into a utilities class--but I'd prefer we not have to maintain it since we'd need to then ensure it works for new browsers/iOS releases. Instead I opted to provide an example in the documentation and reference the [StackOverflow thread](http://stackoverflow.com/questions/9038625/detect-if-device-is-ios).